### DISCUSSION TITLE
Laragon 6.0 and PHP 8 - 32Bits

### DISCUSSION THREAD

**Discussion by loomy2021** [leokhoa/laragon#554] [General]
HI all, is there a way to run php8.2 32 bit on laragon? I've changed the php version but when I try to start the webserver it tells me some different errors (sometims vcruntime140.dll not found, somteimes 0x00007 php-cgi couldn't start) What is the best way to get a server with up2date Laravel running but I need php 32bits because of some external moduls which can only run with php 32 bits :(. Thanks a lot!

**Comment by Pen-y-Fan** [comment]
Yes, you need PHP 8.2 x86, Apache x86 (if running the thread-safe version of PHP) and Visual C++ Redistributable Visual Studio 2015-2022 x86. See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/)

Apache and vc redist x86 can be downloaded from [Apache lounge ](https://www.apachelounge.com/download/VS16/)

**Reply by loomy2021** [reply]
Thank you Pen-y-Fan, works as expected! :)
